docs/api/django.md

```diff
@@ -33,3 +33,16 @@ Returns `True` if a flag is disabled by passing the current request to its condi
   </div>
 {% endif %}
 ```
+
+
+## Passing additional arguments
+
+For some flag conditions, you may want to pass in additional keyword arguments.
+For example, you could pass in the `page` object:
```

Comment:
Tiny nit, "you could pass in a `page` object" would read better, to me. It's less Wagtail-specific.

I kinda recommend re-wording this entirely to something like "Some conditions take additional keyword arguments. For example, you could pass a `page` object:". That way it's clear that whether you *need* to pass in extra keyword arguments depends on the conditions you expect for a particular flag check.